src/asm.rs

```diff
@@ -39,6 +39,52 @@ pub fn wdr() {
     }
 }
 
+/// Get the stack size in bytes, for debbuging.
+#[inline(always)]
+pub fn get_stack_size() -> usize {
```

Comment:
Please mark `get_stack_size()` with `#[cfg(feature = "rt")]`.  This is necessary because we must not make assumptions about the "runtime" (which includes the linker script and its symbols in this case) when `rt` is not selected.

Reply:
Good catch. It's done.